applications/vessel-history/src/features/settings/settings.hooks.ts

```diff
@@ -54,20 +54,17 @@ export const useSettingsConnect = () => {
   }
 }
 
-export const useSettingsRegionsConnect = (
-  section: SettingEventSectionName,
-  settings: SettingsEvents
-) => {
+const onlyUnique = (value: Region, index: number, self: Region[]) => {
```

Comment:
👍  Right, we don't need to create a new instance of this on every call to the `useSettingsRegionsConnect` hook